src/components/Content/Content.js

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import { Eye, MessageCircle, Heart, Image, Paperclip, Video  } from 'react-feather';
+import './Content.css';
+
+function Content() {
+  return (
+  <main>
+    <section className="cards">
+      <div className="card">
+        <div className="card_image-container">
+          <img src="https://images.unsplash.com/photo-1520190282873-afe1285c9a2a?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1200&q=80" alt="cabin ontop of mountain"
+          />
+          <div class="overlay">
+            <span className="text">Mountain retreats</span>
+          <button type="button"><a href="save-button" class="save-button">Save</a></button>
+          </div>
+        </div>
+        <div className="card_content">
+          <div className="icon-wrapper">
+            <span className="image-icon"><a href="#"><Image /></a></span>
+          </div>
+          <div className="card_info">
+            <img src="/afterglow-logo.png" alt="designer logo" className="designer-logo"></img>
+            <span className="text--medium">Outsourced<sup className="company-info">PRO</sup></span>
+                <span className="card_engagement text--medium"><a href="#"><Eye /></a>4910</span>
+                <span className="card_engagement text--medium"><a href="#"><MessageCircle /></a>27</span>
+                <span className="card_engagement text--medium"><a href="#"><Heart /></a>543</span>
```

Comment:
Only the icon becomes clickable when wrapping only the icons in `a` tags. This reduces the amount of "real-estate" the user has to click on in order to trigger the click making it more difficult to click this "engagement". Especially on small devices with touchscreens.

You users will benefit from having the `a` tags as the highest level tag in this component:

```html
<span className="text--medium">Outsourced<sup className="company-info">PRO</sup></span>
    <a href="#" className="card_engagement text--medium">
        <Eye />
        <span>4910</span>
    </a>
    <!-- etc. etc. -->
</span>
```